drivers/wifi/eswifi/eswifi_socket_offload.c

```diff
@@ -392,7 +392,7 @@ static int eswifi_socket_open(int family, int type, int proto)
 	eswifi_lock(eswifi);
 
 	/* Assign dummy context SOCkEt(50CE) */
-	idx = __eswifi_socket_new(eswifi, family, type, proto, 0x50CE);
+	idx = __eswifi_socket_new(eswifi, family, type, proto, (void *)0x50CE);
```

Comment:
What is this magic value? Could you use here `INT_TO_POINTER()` instead?

Reply:
@jukkar socket offloading maintains the zephyr's net_context struct and translates into AT communication to use the TCP/IP stack on chip. ISM43362-M3G-L44-E specifically can handle maximum 4 simultaneous socket connection and it's pre-allocated. 0x50CE initialized to net_context when new socket is created/opened and when more than 4 connections needed, open/socket needs to wait until it gets one free slot.

There is no special meaning to this 0x50CE magic value apart from it, so can be placed to macro and use INT_TO_POINTER. I will change it now.